centreon/www/class/config-generate/generate.class.php

```diff
@@ -303,6 +305,14 @@ public function resetObjectsEngine(): void
             Backend::getInstance($this->dependencyInjector),
             $readAdditionalConnectorRepository
         ))->reset();
+        $readAgentConfigurationRepository = $kernel->getContainer()->get(
+            ReadAgentConfigurationRepositoryInterface::class
```

Comment:
if you need this class, you can pass it in the constructor with a dependencie injection

Reply:
ok